src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -416,12 +458,34 @@ private void registerListeners() {
             refreshGrid(sampleGrid);
             Notification.show("Refreshed");
         });
+
+        cancelImageLoadingButton.addClickListener(event -> {
+
+            LOG.info("--> Warning: about to interrupt loading thread ...");
+
+            imageLoadingThread.interrupt();
```

Comment:
Catch exceptions from connection to omero server when thread dies, provide user (log) with a meaning full explanation of error